src/components/codeContext.tsx

```diff
@@ -96,56 +97,59 @@ const formatApiUrl = ({ scheme, host }: Dsn) => {
   return `${scheme}${apiHost}/api`;
 };
 
-export function fetchCodeKeywords() {
-  return new Promise(resolve => {
-    function transformResults(projects: ProjectApiResult[]) {
-      if (projects.length === 0) {
-        console.warn("Unable to fetch codeContext - using defaults.");
-        resolve(DEFAULTS);
-      } else {
-        resolve({
-          PROJECT: projects.map(project => {
-            const parsedDsn = parseDsn(project.dsn);
-            return {
-              DSN: project.dsn,
-              PUBLIC_DSN: project.dsnPublic,
-              PUBLIC_KEY: parsedDsn.publicKey,
-              SECRET_KEY: parsedDsn.secretKey,
-              API_URL: formatApiUrl(parsedDsn),
-              PROJECT_ID: project.id,
-              PROJECT_SLUG: project.projectSlug,
-              ORG_ID: project.organizationId,
-              ORG_SLUG: project.organizationSlug,
-              ORG_INGEST_DOMAIN: `o${project.organizationId}.ingest.sentry.io`,
-              MINIDUMP_URL: formatMinidumpURL(parsedDsn),
-              UNREAL_URL: formatUnrealEngineURL(parsedDsn),
-              title: `${project.organizationSlug} / ${project.projectSlug}`,
-            };
-          }),
-        });
-      }
+/**
+ * Fetch project details from sentry
+ */
+export async function fetchCodeKeywords() {
+  let json: { projects: ProjectApiResult[] } | null = null;
+
+  const url = process.env.NODE_ENV
```

Comment:
Would NODE_ENV be truthy during production builds?

Reply:
Whoops, that was supposed to be a string comparison :)

Reply:
Ok fixed this guy